indico/modules/events/registration/controllers/__init__.py

```diff
@@ -36,3 +39,37 @@ def _checkParams(self):
                         .find(id=request.view_args['reg_form_id'], is_deleted=False)
                         .options(defaultload('form_items').joinedload('children').joinedload('current_data'))
                         .one())
+
+
+class RegistrationEditMixin(object):
+
+    normalize_url_spec = {
+        'locators': {
+            lambda self: self.registration
+        }
+    }
+
+    def _process(self):
+        form = make_registration_form(self.regform, management=self.management, registration=self.registration)()
+
+        if form.validate_on_submit():
+            modify_registration(self.registration, form.data, self.event)
+            return self._redirect()
```

Comment:
not a good name. `RH` has an ancient legacy method with that name, which has nothing to do with this one